content/20.treatments.md

```diff
@@ -364,6 +364,12 @@ Additionally, one study identified an increased risk of mortality in older men r
 HCQ use for COVID-19 also leads to shortages for anti-malarial or anti-rheumatic use, where it has been definitively proven to be effective.
 Further investigation of HCQ in large, rigorous, multi-center clinical trials encompassing different sex and ages is required.
 
+#### Toll-Like Receptor Agonists and Antagonists
```

Comment:
There is some relation between your suggested addition here & the paragraph draft I wrote as a suggested addition for somewhere in therapeutics. If we want to put these together, we'd have to figure out what subheading would cover everything. Also I'm not sure if these would belong in Biologics or Small Molecules (it's sort of a combination). We should be able to figure it out though.  Here's the paragraph draft I wrote which we might be able to use once we've figured out how to incorporate it / if any subheadings need to be added etc. :

Briefly, another type of approach which is being investigated is the possible use of what is termed trained immunity, in particular as elicited by non-SARS-CoV-2 whole-microorganism vaccines (or other microbial stimuli), as a tool which might be shown to generate heterologous protective effects with respect to SARS-CoV-2 susceptibility or severity [doi: 10.1016/j.cell.2020.04.042].
In a recent review [doi: 10.1038/s41577-020-0285-6], trained immunity was defined as forms of memory which are temporary (e.g., months or years, and reversible), displayed by innate immune cells and innate immune features of other cells, displayed as increased responsiveness to future same or heterologous pathogen infection or sometimes decreased responsiveness or immunological tolerance, and established through epigenetic and metabolic mechanisms; the review also notes various ways in which trained immunity may be related to pathology, inflammatory-related diseases, conditions and immunology associated with aging, and cancer. 
One type of stimulus which may induce trained immunity is BCG vaccination; BCG is an acronym for bacillus Calmette-Guerin, a live attenuated strain derived from an isolate of Mycobacterium bovis with the common reason for application being prevention of tuberculosis in humans.
With respect to SARS-CoV-2 specifically, clinical trials in non-SARS-CoV-2-infected adults have been setup to assess the possible efficacy of BCG vaccination, with one set of trials for example having time until first infection as the primary study endpoint, with some trials being for healthcare workers and other trials being for hospitalized elderly adults without immunosuppression who get vaccinated with placebo or BCG at hospital discharge and another set of trials for older adults (>50 years) under chronic care for conditions like hypertension and diabetes.
An article [doi: 10.1016/j.cell.2020.04.042] that very briefly summarizes the setup of these trials also includes various related considerations (related animal and human studies, general safety, specific possible safety considerations with respect to SARS-CoV-2) and here we mention one of the most important considerations. Severe SARS-CoV-2 has been characterized so far to possibly exhibit dysregulated immune responses and whether or when some immune responses are protective or pathogenic is still under research (e.g., [doi: 10.1016/j.chom.2020.04.009; doi: /10.1016/j.chom.2020.05.009; 10.1016/j.cell.2020.04.042; 10.1016/j.chom.2020.05.008]); also, trained immunity itself may sometimes be related to either balanced, or too much or too little, or useful or harmful immunological responses. The article [doi: 10.1016/j.cell.2020.04.042] proposes that trained immunity might lead to an earlier stronger response which could reduce viremia and avoid (possibly with the additional support of another therapeutic administered later) the later detrimental immunopathology seen with severe cases; this is a possibility, but research is required to assess its validity.

Reply:
@rando2 still not sure how we could possibly combine these things exactly, but hopefully getting closer and in the meanwhile putting the intermediate steps here as comments. Anyways, here are some potentially useful sentences I wrote which are related to a bunch of things: (A) what you wrote here (TLR's are kinds of PRR's), (B) what I wrote in the comment above related to the term trained immunity (C) more importantly, some basic background for how/why vaccines work (since right now I think we just go into the details but didn't mention the background) and (D) what adjuvants are which is related to vaccines (C) and also (A) and (B).

We will describe research into vaccines in detail later.
For basic background, during a first exposure to a pathogen innate (firstly) and acquired (along with innate) immune responses are induced and the acquired immune system at the same time forms memory B cells and memory T cells based on this experience so that on a possible future second exposure to the same pathogen an earlier/quicker and stronger acquired (along with the innate) immune response is generated which largely prevents or greatly reduces infection. [need to double-check/maybe rewrite a bit to make sure sounds more like textbook description]
Vaccines are (roughly, with more detail later) attenuated or inactivated pathogens or antigen parts of pathogens which are meant to generate long-term memory of a pathogen or antigens of a pathogen so that if or when a person is exposed to the actual pathogen in the future they will have a strong acquired immunity memory response similar to what occurs upon second exposure to a pathogen.

Adjuvants are sometimes included within vaccines, especially vaccines other than live attenuated and inactivated viruses, in order to enhance the immune response.

A review on the development of SARS-CoV-2 vaccines [doi:10.3390/vaccines8020153] also included a brief summary of considerations for adjuvants to research for the vaccines including a brief description of some already commonly used adjuvants.
Different adjuvants can regulate different types of immune responses, so the type of adjuvant or combination of adjuvants for use in a vaccine depends on the type of vaccine and the research into what works with respect to efficacy and safety.

Adjuvants include a variety of molecules or larger microbial-related products which sometimes are or include immunostimulants or immunomodulators or more generally have an effect on the immune system or immune responses of interest.

A variety of possible mechanisms for adjuvants have been researched [doi:10.1038/nri2510; doi:10.3389/fimmu.2018.02619; doi:10.3390/vaccines8010128], including induction of DAMPs (damage-associated molecular patterns) which can be recognized by certain pattern recognition receptors (PRRs) of the innate immune system, being/including/functioning as PAMPs (pathogen-associated molecular patterns) which can also be recognized by certain pattern recognition receptors (PRRs), and more generally enhancing humoral (type 2) or cellular (type 1) immune responses.
When choosing an adjuvant or combination of adjuvants, considerations include promotion of advantageous effects of the components and immune response and avoiding or inhibiting possible deleterious effects of the components and immune response.
There are also considerations, technologies, and research into the effects of different ways to deliver (or co-deliver or combine) the adjuvant and antigen components of a vaccine.

Plus one more sentence which is sort of related to these topics but I'm not sure if it would just unnecessarily add confusion:
Another illustration of an interesting combination of immune defense and suppression of excess inflammation with altered susceptibility to different pathogen infection is described in a review on regulation of immune response to respiratory infection [doi:10.1038/nri3166]: during the 1918 type A influenza pandemic and the 2009 H1N1 influenza pandemic, secondary bacterial pneumonia was the predominant cause of death; the possible increase and severity of secondary bacterial infections during or shortly after (or for some mechanisms weeks or months after) influenza type A infection is believed to occur because of decreased barrier function of the epithelium due to damage from the viral infection or, also, through other mechanisms in which the innate and adaptive immune cells display an altered response with respect to bacterial infection perhaps as part of immune responses regulatory features to prevent excess pulmonary inflammation along with their functions for controlling viral replication or infection.

Reply:
@rando2 To separate things out a bit, here's the info above just about adjuvants for vaccines. (adjuvants are also related to the other stuff but those are different strategies which in some ways are less understood and have less widespread use than vaccines as typically applied/intended). An adjuvant (sub-)section could be put within the vaccines section. A while back on the gitter-chat it was mentioned to maybe put a section on adjuvants somewhere.  Here's copied just the parts from above that's for adjuvants and could be used as adjuvant subsection under vaccines for example. If we put this in then we can get some more checks/adjustments for it too from other immunology people.
Adjuvants for Vaccines (or maybe just call the subsection "Adjuvants")
Adjuvants are sometimes included within vaccines, especially vaccines other than live attenuated and inactivated viruses, in order to enhance the immune response.
A review on the development of SARS-CoV-2 vaccines [doi:10.3390/vaccines8020153] also included a brief summary of considerations for adjuvants to research for the vaccines including a brief description of some already commonly used adjuvants.
Different adjuvants can regulate different types of immune responses, so the type of adjuvant or combination of adjuvants for use in a vaccine depends on the type of vaccine and the research into what works with respect to efficacy and safety.
Adjuvants include a variety of molecules or larger microbial-related products which sometimes are or include immunostimulants or immunomodulators or more generally have an effect on the immune system or immune responses of interest.
A variety of possible mechanisms for adjuvants have been researched [doi:10.1038/nri2510; doi:10.3389/fimmu.2018.02619; doi:10.3390/vaccines8010128], including induction of DAMPs (damage-associated molecular patterns) which can be recognized by certain pattern recognition receptors (PRRs) of the innate immune system, being/including/functioning as PAMPs (pathogen-associated molecular patterns) which can also be recognized by certain pattern recognition receptors (PRRs), and more generally enhancing humoral (type 2) or cellular (type 1) immune responses.
When choosing an adjuvant or combination of adjuvants, considerations include promotion of advantageous effects of the components and immune response and avoiding or inhibiting possible deleterious effects of the components and immune response.
There are also considerations, technologies, and research into the effects of different ways to deliver (or co-deliver or combine) the adjuvant and antigen components of a vaccine.

Reply:
@rando2 see new comment above on how to organize/separate the related info; the comment above has a possible Adjuvants for Vaccines subsection. Now for the other stuff, here's an idea for how we could put together what you wrote on TLR agonists & what I wrote on trained immunity. Let me know what you think, and then we could ask also for more feedback from other immunologists:
-Subsection title: To do this, it may be more correct (or at least general enough to cover more possibilities) if we rename the draft subsection as something like "Interactions with Innate Immune Response Components" or "Modulators of Innate Immune Response Components" (rather than what you have now as "Toll-like receptor agonists and antagonists"), but we'd need feedback on the subsection title (and everything else).
-We'd need an opening sentence that's more general. Maybe something like: "There is also some research into possible potential therapeutics or prophylactics which interact with components of the innate immune response."
-Then we could put the 2 sentences you wrote about TLR agonists.
-After (or before) your sentences maybe we could add a note like: "More broadly, there are a variety of toll-like receptors which are examples of pattern recognition receptors, innate immune response components which recognize pattern-associated molecular patterns (PAMPs) and damage-associated molecular patterns (DAMPs)." & maybe also add the clarifying note: "Note that Toll-like receptors form a part of innate immune recognition and can more generally contribute to promoting both innate and adaptive responses." (will put in the ref for the latter sentence -- i used it before, just have to find the ISBN again)
-Maybe we could also mention as a note something like: "As a related note, vaccine adjuvants also in part sometimes directly or indirectly interact with various pattern recognition receptors, which are innate immune response components; details related specifically to vaccine adjuvants are described in a separate subsection related to Vaccines."
-Then we could put the stuff I wrote on trained immunity (with the feedback I got from you and anyone else incorporated).
-We then might want to also add the other sort of related note/sentence which I suggested at the end of the comment before the comment before this one (i.e., two comments above)

Reply:
I copied all of this into #496 so that we can see everything as it all connects together! Thank you for this feedback and I think this is a really cool avenue to look at!